connexion/apis/aiohttp_api.py

```diff
@@ -85,6 +102,14 @@ def _get_openapi_json(self, req):
             body=self.jsonifier.dumps(self.specification.raw)
         )
 
+    @asyncio.coroutine
+    def _get_openapi_yaml(self, req):
+        return web.Response(
+            status=200,
+            content_type='text/yaml',
```

Comment:
There are multiple possible content_types for yaml afaict.
I opted for 'text/yaml' but I think `application/vnd+yaml` is another possibility as well.
I don't have a preference, just wanted to note this.

Reply:
I know. I followed your mime though ;)